src/Controllers/GagePageController.ts

```diff
@@ -536,13 +569,287 @@ module StreamStats.Controllers {
                 }
             }
         }
+
+        // Gage Analysis Plots methods
+
+        public getGagePlots() {
+            this.getPeakInfo(); // Annual Peak Streamflow plot
+        }
+
+        //Get peak values from NWIS
+        public getPeakInfo() {
+            const url = 'https://nwis.waterdata.usgs.gov/usa/nwis/peak/?format=rdb&site_no=' + this.gage.code
+            console.log('GetPeakURL', url)
```

Comment:
Comment out or delete all `console.log` statements-- there are a couple still in here